retext.py

```diff
@@ -54,6 +57,39 @@ def main():
 	app.setOrganizationDomain('mitya57.me')
 	if hasattr(app, 'setDesktopFileName'): # available since Qt 5.7
 		app.setDesktopFileName('me.mitya57.ReText.desktop')
+
+	singleApp = SingleApplication("D1278E7822F011E6802800F1F38F93EF")
```

Comment:
OK, It will be changed to "ReText", I just concentrated upon unique value at the beginning.
Later it will be changed to something like "ReText-username" for multi-user that @Griffon26 referred.